flake8_html/plugin.py

```diff
@@ -142,14 +156,33 @@ def finished(self, filename):
                 unique_messages > 1 or any(e[2] > 1 for e in errs),
                 errs
             ))
+            if self.pep8report:
+                # pep8 report - Step 1: gather errors
+                for err in errors:
+                    pep8_report_errs.append(
+                        (
+                            err.filename, err.line_number, err.column_number,
+                            err.code, err.text
+                        )
+                    )
+
+        if self.pep8report:
+            # pep8 report - Step 2: sort gathered errors by
+            # filename, line- and column number
+            pep8_report_errs.sort(key=lambda err: (err[0], err[1], err[2]))
+            # Step 3: Profit. Print the errors in pep8 report format
+            for e in pep8_report_errs:
+                print("%s:%d:%d: %s %s" % e)
```

Comment:
And here you could write something readable like

    print("{e.filename}:{e.line_number}:{e.column number} {e.code} {e.text}".format(e=e))